### ISSUE TITLE
multiple nodes for Glatimer

### ISSUE THREAD

**Issue by TranslatorIssueCreator** [NCATSTranslator/Feedback#576]
## Steps to reproduce:
MVP1 for multiple sclerosis
filter for Glatimer

**Comment by gglusman** [comment]
'Glatimer' finds nothing, but 'Glatiramer' does indeed find duplicate results.
1st 'glatiramer': http://identifiers.org/unii/U782C039QP
2nd 'glatiramer': http://identifiers.org/pubchem.compound/65370
1st 'glatiramer acetate': http://identifiers.org/chembl.compound/CHEMBL1201507
2nd 'glatiramer acetate': http://identifiers.org/pubchem.compound/3081884

**Comment by gglusman** [comment]
Noting also one result for 'dexamethasone' and two for 'dexamethasone sodium phosphate'.

**Comment by gglusman** [comment]
Actually there's plenty of duplicates. Even the top result 'mitoxanthrone' is returned six times, three of which have the exact same name 'Mitoxanthrone', one is 'Mitoxanthrone hydrochloride' and the other two have UNII and Chembl.compound curies displayed.

**Comment by sandrine-muller-research** [comment]
On Glatimer:
[NameRes does not output anything](https://name-resolution-sri-dev.apps.renci.org/lookup?string=Glatimer&offset=0&limit=10) for this compound
[NodeNorm does not normalize](https://nodenormalization-sri.renci.org/1.4/get_normalized_nodes) UNII and drug bank IDs

**Comment by cbizon** [comment]
Yeah, pretty strange.  At prod nodenorm, none of the Mitoxanthrone identifiers seem to be returning anything but null.  @gaurav could there be a load problem for prod NN?

On dev nn, Mitoxanthrone is nicely normalized, and if you use the chemical conflation, it even pulls in the mitoxanthrone hydrochloride.

**Comment by gaurav** [comment]
> Noting also one result for 'dexamethasone' and two for 'dexamethasone sodium phosphate'.

This also appears to be fixed with drug-chemical conflation (see [NameLookup CI result](https://name-lookup.ci.transltr.io/lookup?string=dexamethasone&autocomplete=true&offset=0&limit=10)), but we're calling this conflated clique "dexamethasone acetate" for some reason. I've filed that as https://github.com/TranslatorSRI/Babel/issues/318

**Comment by sstemann** [comment]
The issue for Glatiramer still exists in Hammerhead 

https://ui.test.transltr.io/results?l=Multiple%20Sclerosis&i=MONDO:0005301&t=0&r=0&q=bb83a3ba-c089-4529-9311-c832c7e8ad42

One result is https://precision.fda.gov/uniisearch/srs/unii/U782C039QP
- this one actually links to the pubchem below

Another result is:  https://pubchem.ncbi.nlm.nih.gov/compound/65370

Dexamethasone seems fine

No results match "Mitoxanthrone"

One result matches "Mitoxantrone"

So I think the issue with Glatiramer still exists @gaurav 

![image](https://github.com/user-attachments/assets/9763f3c0-a141-4ba8-95bc-1855c57673c2)
